### PR TITLE
Readme update for local dev minikube

### PR DESCRIPTION
**Pull request by dalehille**

### CODE REVIEW

File: MINIKUBE.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# local development with minikube
+
+1. clone and run [razeedash](https://github.com/razee-io/Razeedash.git)
+
+```bash
+    git clone razeedash
```
Comment (mckaymic) [inline MINIKUBE.md]: Shouldn't this be `git clone https://github.com/razee-io/Razeedash.git`?

File: MINIKUBE.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    cd razeedash
+    meteor npm install
+    meteor
+```
+
+1. clone and run [razeedash-api](https://github.com/razee-io/Razeedash-api.git)
+
+```bash
+    git clone razeedash-api
```
Comment (mckaymic) [inline MINIKUBE.md]: Similar comment from above

File: MINIKUBE.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
+# local development with minikube
+
+1. clone and run [razeedash](https://github.com/razee-io/Razeedash.git)
+
+```bash
```
Comment (geoffcorey) [inline MINIKUBE.md]: Needs to be indented 2 tabs so ordered number steps will increment on the markdown

File: MINIKUBE.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    git clone https://github.com/razee-io/Razeedash.git
+    cd razeedash
+    meteor npm install
+    meteor
+```
+
+1. clone and run [razeedash-api](https://github.com/razee-io/Razeedash-api.git)
+
+```bash
```
Comment (geoffcorey) [inline MINIKUBE.md]: Needs to be indented 2 tabs so ordered number steps will increment on the markdown

File: MINIKUBE.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+    git clone https://github.com/razee-io/Razeedash-api.git
+    cd razeedash-api
+    npm install
+    npm start
+```
+
+1. edit your hosts file (/etc/hosts on mac and linux) and add:
+
+```bash
```
Comment (geoffcorey) [inline MINIKUBE.md]: Needs to be indented 2 tabs so ordered number steps will increment on the markdown

File: MINIKUBE.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+    192.168.99.1 minikube-host
+```
+
+1. [Install minikube on your laptop](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+
+For macs:
```
Comment (geoffcorey) [inline MINIKUBE.md]: Needs to be indented 2 tabs so ordered number steps will increment on the markdown

File: MINIKUBE.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+    ⌛  Waiting for SSH access ...
+    🐳  Configuring environment for Kubernetes v1.14.2 on Docker 18.09.6
+    🔄  Relaunching Kubernetes v1.14.2 using kubeadm ...
+    ⌛  Verifying: apiserver proxy etcd scheduler controller dns
+    🏄  Done! kubectl is now configured to use "minikube"
+```
+
+1. Configure minikube
+
+- Make sure your KUBECONFIG is pointing to your minikube cluster
```
Comment (geoffcorey) [inline MINIKUBE.md]: Needs to be indented 2 tabs so ordered number steps will increment on the markdown

File: MINIKUBE.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+- `kubectl create namespace razee`
+- kubectl apply this yaml:
+
+```yaml
+    kind: Service
```
Comment (geoffcorey) [inline MINIKUBE.md]: Snippet has an extra space after 1st line when cutting and pasting to file that makes the indents messed up.  Can that be fixed?

File: MINIKUBE.md [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+    apiVersion: v1
+    metadata:
+        name: minikube-host
+        namespace: razee
+    spec:
+        type: ExternalName
+        externalName: minikube-host
+```
+
+1. Access the [welcome screen](http://localhost:3000) of your RazeeDash instance.
```
Comment (geoffcorey) [inline MINIKUBE.md]: Missing the step to actually install Razeedash

**Pull request merged**